Client/src/js/views/SchoolBuses.jsx

```diff
@@ -259,7 +260,17 @@ var SchoolBuses = React.createClass({
   },
 
   email() {
+    this.setState({ showEmailDialog: true });
+  },
 
+  sendEmail(email){
+    Api.sendEmail(email).finally(() => {
+      this.setState({ showEmailDialog: false });
```

Comment:
Two comments:

- On successful send, it looks like the email dialog will just close.  We should notify the user that the email was sent successfully.  A simple notification (Email sent successfully <<OK>>) followed by the dialog closing is all that is needed.
- On failure, it looks like the window will just close without a notice to the user.  This is likely to cause confusion.  We should present an error message and provide the user the opportunity to retry or cancel and return to the dialog.  In this case, the Email Dialog should remain open until the user selects to close the Email Dialog.